### PR TITLE
Create a container for kernel module building

### PR DESCRIPTION
**Pull request by dcermak**
For https://jira.suse.com/browse/PED-4822

### CODE REVIEW

**Review by dirkmueller — COMMENTED** [review]
There's more needed I'd think. Either we need an extra repository added or at least package the ga versions of the kernel packages?

File: src/bci_build/package.py [hunk 1]
```diff
@@ -2564,6 +2564,18 @@ def _get_minimal_kwargs(os_version: OsVersion):
     for os_version in ALL_NONBASE_OS_VERSIONS
 ]
 
+KERNEL_MODULE_CONTAINERS = [
+    OsContainer(
+        name="kernel-module-toolkit",
```
Comment (dirkmueller) [inline src/bci_build/package.py:2569]: Can we name it a bit more specific? I think something like sle15 should be in the name to make it clear that its not for ANY kernel

File: src/bci_build/package.py [hunk 2]
```diff
@@ -2564,6 +2564,18 @@ def _get_minimal_kwargs(os_version: OsVersion):
...
+        pretty_name="Kernel Module Toolkit",
+        package_name="kernel-module-toolkit",
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        package_list=["kernel-devel", "kernel-syms", "gcc"],
```
Comment (dirkmueller) [inline src/bci_build/package.py:2574]: %kernel_module_package_buildreq is expanding to `kmod-compat kernel-syms kernel-syms-azure` in SLE15. so I think we're missing

Comment (dcermak) [reply src/bci_build/package.py:2840]: Do we really want to include the kernel symbols for the Azure kernel? That appears a bit out of place here.

**Review by dirkmueller — APPROVED** [review]

**Pull request merged**